### PR TITLE
feat: create Route53 zone for Serverless API

### PR DESCRIPTION
**Pull request by patheard**
# Summary 
Creates a Route53 zone that will be used to manage the Platform MVP domain.

### CODE REVIEW

File: terragrunt/env/dev/domain/terragrunt.hcl [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "../../../aws//domain"
```
Comment (timarney) [inline terragrunt/env/dev/domain/terragrunt.hcl:7]: Extra slash here?

```
//domain
```

**Review by CalvinRodo — APPROVED** [review]
:taco::cat:

File: terragrunt/env/dev/domain/terragrunt.hcl (hunk 1 above)
Comment (CalvinRodo) [reply terragrunt/env/dev/domain/terragrunt.hcl:7]: This is how it works. 

https://github.com/cds-snc/covid-alert-metrics-terraform/blob/main/env/staging/s3/terragrunt.hcl

Comment (patheard) [reply terragrunt/env/dev/domain/terragrunt.hcl:7]: You would think that, but it's one of the oddities of Terragrunt that allows it to resolve relative paths.

**Review by timarney — APPROVED** [review]

**Pull request merged**